<launch>
  <!-- Define arguments with default values -->
  <arg name="bag1_topic" default="/rvr/odom" />
  <arg name="bag2_topic" default="/tf" />
  <arg name="frame_id" default="base_link_31" />
  <arg name="bag1" default="/home/said/fxpbags/rvr1/rwrvr1_stand_7.bag" />
  <arg name="bag2" default="/home/said/fxpbags/tracking/trackrw.bag" />

  <!-- Pass arguments as parameters to the node -->
  <node pkg="mercator_sensor_fusion_ros_pkg" type="rosbag_syncer" name="rosbag_syncer" output="screen"  >
    <param name="bag1_path" value="$(arg bag1)" />
    <param name="bag2_path" value="$(arg bag2)" />
    <param name="bag1_topic" value="$(arg bag1_topic)" />
    <param name="bag2_topic" value="$(arg bag2_topic)" />
    <param name="frame_id" value="$(arg frame_id)" />
  </node>
</launch>
